Bottom: bdaf5dbe27f31490ee0eaac9e8f10acc3e761134
Top:    08f10b282a74b053a93f5b3b3a811689f9a71a90
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 15:34:33 -0500

Use internal mail server


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index 1ac46a1a..55e678f8 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -53,14 +53,21 @@ mn = reporters.MailNotifier(
     sendToInterestedUsers=True,
     extraRecipients=["andrew.cooper@novatechweb.com"],
     useTls=True,
-    relayhost=os.getenv('BUILDBOT_SMTP_HOST',"relay.dnsexit.com"),
-    smtpPort=26,
-    smtpUser=os.getenv('BUILDBOT_SMTP_USER', "Dean.cox"),
-    smtpPassword=os.getenv('BUILDBOT_SMTP_PASS',"no5156")
+    relayhost=os.getenv('BUILDBOT_SMTP_HOST'),
+    smtpPort=os.getenv('BUILDBOT_SMTP_PORT'),
+    smtpUser=os.getenv('BUILDBOT_SMTP_USER'),
+    smtpPassword=os.getenv('BUILDBOT_SMTP_PASS')
     )
 c['services'].append(mn)
 
-####### WEB SERVER
+gl = reporters.GitLabStatusPush(
+    os.getenv('GITLAB_TOKEN'),
+    context="Buildbot CI",
+    baseURL=os.getenv("GITLAB_URI"),
+    verbose=True,
+    debug=True
+)
+c['services'].append(gl)
 userInfoProvider = util.LdapUserInfo(
         uri=os.getenv('LDAP_URI'),
         bindUser=os.getenv('LDAP_BIND_USER'),
